### PR TITLE
HotStrip  algorithm updated

### PR DESCRIPTION
**Pull request by suchandradutta**
updating Hot Strip algorithms in CaliTracker/SiStripQuality putting extra requirement on the strip Digi wrt APV median as developed by Mario Galanti + Message logger ourput made more logical in DQMOffline/CalibTracker/plugins/SiStripQualityHotStripIdentifierRoot.cc

### CODE REVIEW

**Comment by mmusich** [comment]
@cmsbuild please test

File: CalibTracker/SiStripQuality/src/SiStripBadAPVandHotStripAlgorithmFromClusterOccupancy.cc [hunk 1]
```diff
@@ -4,12 +4,14 @@
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "DataFormats/DetId/interface/DetId.h"
 #include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "Geometry/Records/interface/IdealGeometryRecord.h"
```
Comment (mmusich) [inline CalibTracker/SiStripQuality/src/SiStripBadAPVandHotStripAlgorithmFromClusterOccupancy.cc]: @suchandradutta can you clarify why is this needed now?

**Comment by suchandradutta** [comment]
It was included mistakenly! I have removed it

**Comment by mmusich** [comment]
@suchandradutta thank you very much

**Comment by mmusich** [comment]
@cmsbuild please acknowledge changes

**Comment by mmusich** [comment]
please test

**Comment by deguio** [comment]
+1

**Comment by mmusich** [comment]
+1

**Comment by davidlange6** [comment]
+1
